ios/brave-ios/Sources/AIChat/Components/Settings/AIChatAdvancedSettingsView.swift

```diff
@@ -351,9 +351,40 @@ public struct AIChatAdvancedSettingsView: View {
           )
         }
       }
+
+      if swipedLeftCount >= 3 && swipedRightCount >= 3 && swipedUpCount >= 1 && swipedDownCount >= 1
+      {
+        Section {
+          NavigationLink {
+            AIChatLeoPurchaseLogs(model: model)
+          } label: {
+            LabelView(
+              title: "View Leo Purchase Logs",
+              subtitle: model.models.first(where: { $0.key == model.defaultAIModelKey })?
+                .displayName
+                ?? model.currentModel.displayName
+            )
+          }.listRowBackground(Color(.secondaryBraveGroupedBackground))
+        } header: {
+          Text("Leo Purchase Logs")
+            .textCase(nil)
+        }
+      }
```

Comment:
Make `AIChatLeoPurchaseLogs` public instead, place it in SettingsViewController debug section